src/main/resources/static/domain-events-api.yml

```diff
@@ -1410,7 +1410,6 @@ components:
         - assessmentId
         - assessmentUrl
         - personReference
-        - allocatedBy
```

Comment:
Is this removed by mistake?

Reply:
I'd say that an allocation that happens automatically isn't "by" anyone, so would best be represented by `null`.